### PR TITLE
Update ansible_min_version

### PR DESCRIPTION
**Pull request by nickovivar**
This PR updates the min_ansible version that it's necessary to run this role. The ansible_min_version is 2.6.0 that is the minor supported version.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: Base image and common roles
   company: Stack Builders
   license: MIT
-  min_ansible_version: 2.4.0
+  min_ansible_version: 2.6.0
```
Comment (sestrella) [inline meta/main.yml]: @nickovivar I think you also need to get rid of the following files:
- https://github.com/stackbuilders/sb-debian-base/blob/master/requirements-2.4.txt
- https://github.com/stackbuilders/sb-debian-base/blob/master/requirements-2.5.txt

Comment (sestrella) [reply meta/main.yml]: @nickovivar could you also update the CHANGELOG? Additionally, is it a backward-incompatible change? Or it just produces a warning when trying to run this role with an old Ansible version

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 * Remove package emacs24-nox ,to work arround issue #114
 * Add Debian 10 support
 * Drop Ubuntu 14.04 support
+* Drop Ansible 2.4.0 and 2.5.0 support
```
Comment (sestrella) [inline CHANGELOG.md]: @nickovivar `v1.6.2` is already a released version, I think you need to bump the version here.

File: meta/main.yml (hunk 1 above)
Comment (sestrella) [reply meta/main.yml]: @nickovivar I also found there is a section in the [README.md](https://github.com/stackbuilders/sb-debian-base/blob/master/README.md) file that talks about Ansible minimum version:

```diff
- Generic Debian image for servers. This Galaxy requires Ansible 2.4.0
+ Generic Debian image for servers. This Galaxy requires Ansible 2.6.0
```

Could you please update that section too?

**Comment by nickovivar** [comment]
@sestrella I made a check with Ansible 2.4 and 2.5 and there's no errors and warnings during the installation.
Screenshot from 2019-11-11 13-01-16
Screenshot from 2019-11-11 13-06-41

So I update the changelog to update the version to 1.7.0.

**Pull request merged**